maat-court-data-api/src/main/java/gov/uk/courtdata/iojappeal/controller/IOJAppealControllerV2.java

```diff
@@ -41,25 +33,22 @@
 @RequestMapping("${api-endpoints.assessments-domain-v2}/ioj-appeals")
 public class IOJAppealControllerV2 {
 
-    private final IOJAppealService iojAppealService;
+    private final IOJAppealV2Service iojAppealService;
```

Comment:
You may be following existing convention within MAAT API here, in which case that's fine (lest we want to refactor everything), but in one of @JHunt16 's recent PRs he has changed the naming of `IOJ` to be camel case also, making it `Ioj` for class names / method names / variables etc and `IoJ` for string descriptions. If it doesn't go against everything already in MAAT API then it'd be nice to do the same here 🙂